stl/inc/limits

```diff
@@ -11,6 +11,8 @@
 #include <cfloat>
 #include <climits>
 #include <cwchar>
+#include <intrin.h>

```

Comment:
Same comment about `intrin.h` being extremely expensive - dragging it into `<bit>` is one thing, but `<limits>` is widely included.